basic-helm-spring-boot/.helm/jenkins-agent-helm/templates/buildconfig.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  labels:
+    application: {{ .Release.Name }}
+  name: {{ .Release.Name }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  output:
+    to:
+      kind: ImageStreamTag
+      name: {{ .Release.Name }}:{{ .Values.destTag }}
+  source:
+    type: Git
+    git:
+      uri: {{ .Values.uri }}
+      ref: {{ .Values.ref }}
+    contextDir: {{ .Values.contextDir }}
+  strategy:
+    dockerStrategy:
+      dockerfilePath: {{ .Values.dockerfilePath }}
+    type: Docker
+  triggers: []
```

Comment:
A `ConfigChange` and/or `ImageChange` trigger should be configured here which could automatically trigger a build instead of requiring the user to manually start the build